### ISSUE TITLE
Standalone.py start location affects functionality

### ISSUE THREAD

**Issue by mfarrugi** [nlplab/brat#1131]
If a script runs standalone.py, e.g.

```
#!/bin/csh -f
python brat/standalone.py >>& httpReq.log
```

You get various 403 / 404 errors when attempting to use the application.

This isn't particularly high priority, but this issue could help anyone else that runs into the same issue w/ Standalone.py

**Comment by icedwater** [comment]
Agreed, I don't seem to be able to access the server if it starts from outside where `standalone.py` is. For your case, maybe this would be a fix:

    #! /bin/csh -f
    python standalone.py >>& ../httpReq.log

i.e. start from inside the script's folder? Yes, I know it's over 4 years old at this point...
